content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -94,7 +94,7 @@ For a reference to old feature gates that are removed, please refer to
 | `DownwardAPIHugePages` | `false` | Beta | 1.21 | 1.21 |
 | `DownwardAPIHugePages` | `true` | Beta | 1.22 | |
 | `DynamicResourceAllocation` | `false` | Alpha | 1.26 | |
-| `EventedPLEG` | `false` | Alpha | 1.26 | - |
```

Comment:
You are not supposed to remove the old line 97. It was and still is about the Alpha stage.
You can revise it to indicate that Alpha ends at 1.26 as well.

**after** this revision, you can add a new line for Beta stage as you'd already done.